rfc6962-bis.xml

```diff
@@ -766,7 +776,7 @@ messages.
           <t hangText="error_code:">
             An error code readable by the client. Some codes are generic and are detailed here. Others are detailed in the individual requests. Error codes are fixed text strings.
             <list style="hanging">
-              <t hangText="not compliant">
+              <t hangText="'not compliant'">
```

Comment:
In a text copy it's hard to spot where the error code string ends and the text describing the error code begins, so I think some sort of delimiter would be useful.